easybuild/easyconfigs/d/DMTCP/DMTCP-2.6.0-20210326.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'DMTCP'
+version = '2.6.0'
+local_commit = 'd0cec6b2ce32201e6b4de6a478d3f3f4032ac121'
+local_commit_date = '20210326'
+versionsuffix = '-%s' % (local_commit_date)
```

Comment:
Normally, if we need to do this, we could call the version just the date, as it's really not the 2.6.0 version anymore (quite far from it).

Reply:
Except this is from the 2.6 branch and not from the main branch (which is sort of 3.x). This commit has bug fixes for 2.6 which are not included in that 2.6.0 release tarball